src/graph_gui.cpp

```diff
@@ -348,6 +348,7 @@ struct BaseGraphWindow : Window {
 		GfxFillRect(r.left, r.top, r.right, r.bottom, GRAPH_BASE_COLOUR);
 
 		/* Draw the vertical grid lines. */
```

Comment:
Nitpick: this comment was already weird, and it is now even weirder. The next statement does not draw the vertical grid lines :D Maybe another newline? Dunno .. it is just a weird comment .. the rest should have been in a function looking at the comment :P